assets/js/amber.js

```diff
@@ -226,3 +226,18 @@ export class Socket {
 module.exports = {
   Socket: Socket
 }
+
+
+/**
+ * Allows delete links to post for security and ease of use similar to Rails jquery_ujs
+ */
+$(document).ready(() => {
+  $( "a[data-method='delete']" ).click(function(){
+    if(confirm("Are you sure you want to delete this?")){
```

Comment:
What if someone would like to display a different message? I thought using `button_to` would have been lesser of a change even if styling was required since the bootstrap theme we should is just that a bootstrap.

Reply:
It would use data-message or something like rails. I ran out of time for that last night and my son is in surgery right now so I'm afk.

Reply:
Displaying a different message is a good idea, but doesn't seem like it should be required to get this feature in.